misk/src/main/kotlin/misk/metrics/backends/stackdriver/StackDriverBackendConfig.kt

```diff
@@ -0,0 +1,11 @@
+package misk.metrics.backends.stackdriver
+
+import misk.config.Config
+import java.time.Duration
+
+data class StackDriverBackendConfig(
+        val project_id: String,
+        val service_path: String,
+        val root_url: String,
+        val interval: Duration,
+        val batch_size: Int = 200) : Config
```

Comment:
nit: give each field its own line and move the `) : Config` bit to its own line.